chrome/content/zotero/zoteroPane.xul

```diff
@@ -110,6 +110,8 @@
 								<menupopup>
 									<menuitem id="zotero-tb-feed-add-fromURL" label="&zotero.toolbar.feeds.new.fromURL;" 
 										command="cmd_zotero_newFeed_fromURL"/>
+									<menuitem id="zotero-tb-feed-add-fromOPML" label="&zotero.toolbar.feeds.new.fromOPML;"
+											  oncommand="ZoteroPane_Local.importFeedsFromOPML()"/>
```

Comment:
It probably won't matter, since there's a good chance 5.0 will be Standalone only and "From Page…" will be removed, but in the meantime I think this should go at the bottom, since it should be less common than "From Page…"